downstream/modules/eda/proc-eda-set-up-token.adoc

```diff
@@ -24,7 +24,7 @@ For more information about creating the token, see the link:https://docs.ansible
 +
 [NOTE]
 ====
-The token must be in write-scope.
+The token you create must have write permissions, and the assigned user must have permission to execute job and workflow templates.
```

Comment:
"job template" is a different entity than "workflow template". Is maybe better to not omit the word "template" to avoid misleading? 

```suggestion
The token you create must have write permissions, and the assigned user must have permission to execute job templates and workflow templates.
```

Reply:
@Alex-Izquierdo Can we get a screen shot of the minimal permission from the AAP UI so we don't have any ambiguities.

Reply:
I think that is a call for @jself-sudoku 
In the existing documentation we don't have specific instructions about controller, we just point to a link of their documentation. 

If you want to add a screenshot I can take it for you.

Reply:
@Alex-Izquierdo - I didn't delete a word there. I combined job templates and workflow templates, which is a grammatical and style rule. Instead of saying template twice, we just say job and workflow templates. 

@mkanoor - all docs teams are encouraged to use as few screenshots as possible due to the work it takes to maintain them, specifically if screen shots change. Can you let me know what ambiguities you're referring to in this instance so we can minimize that?

Reply:
@Alex-Izquierdo and @mkanoor - Just an FYI. The docs team request SME reviews for technical accuracy. Editorial and peer reviews focus more on grammar and style. We usually handle those after we ensure that the content is technically accurate. Can one or both of you provide approval so that we can move this quick change to the next phase of our processes and then ensure that the change is also applied for 2.5?

Reply:
@Alex-Izquierdo @jself-sudoku I don't know how to create tokens, there is an older version of the UI and a newer version. I tried looking in the new UI and didn't see an option under Users to create a Token. Is this different in the newer UI?

Reply:
Hi @mkanoor. I ran into the same issue, which is why I asked the question about it in the Slack discussion [here](https://redhat-internal.slack.com/archives/C04930GDCTS/p1716426880812629). You can see my question [here](https://redhat-internal.slack.com/archives/C04930GDCTS/p1716931145237329?thread_ts=1716426880.812629&cid=C04930GDCTS), which is now crossed out since my teammate, @dcdacosta , shared a link to the latest [mockups]([https://www.figma.com/proto/gMrHPRqYwvZw5yrkL8tXHF/Access-Management?page-id=82%3A8[…]port=1257%2C-3512%2C0.61&t=Yzn8joBu2pzCzvui-1&scaling=min-zoom](https://www.figma.com/proto/gMrHPRqYwvZw5yrkL8tXHF/Access-Management?page-id=82%3A89874&type=design&node-id=1690-82285&viewport=1257%2C-3512%2C0.61&t=Yzn8joBu2pzCzvui-1&scaling=min-zoom)). There you can see what is being planned to carry out this process for 2.5. The tab is simply referred to as **Tokens** in the mockups.